### PR TITLE
Update components for k8s 1.15.5 release

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/6431

### CODE REVIEW

**Comment by xh3b4sd** [comment]
This PR still runs all version bundles. This means all versions are reconciled. This means with VOO where ever you deploy this you cause conflicts within the environment because multiple operators reconcile the same CR. This breaks clusters. Please make sure to merge with master and do not deploy anything that causes such conflicts.

File: Gopkg.toml [hunk 1]
```diff
@@ -186,4 +186,4 @@ required = [
 
 [[prune.project]]
   name = "github.com/giantswarm/k8scloudconfig"
-  unused-packages = false
+  unused-packages = false
```
Comment (MarcelMue) [inline Gopkg.toml]: Can you fix this with your editor to have a newline at the end of files.

File: README.md [hunk 2]
```diff
@@ -148,3 +148,4 @@ for details.
 ## Credit
 - https://golang.org
 - https://github.com/giantswarm/microkit
+
```
Comment (MarcelMue) [inline README.md]: I think we added this newline for testing earlier.

File: Gopkg.toml (hunk 1 above)
Comment (tfussell) [reply Gopkg.toml]: Done.

File: README.md (hunk 2 above)
Comment (tfussell) [reply README.md]: Good catch. Fixed.

**Pull request merged**